packages/core/config/storybook/main.js

```diff
@@ -17,8 +15,13 @@ module.exports = {
       : require('../webpack.development')
 
     // We replace imports to "@redwoodjs/router" with our own implementation in "@redwoodjs/testing"
-    sbConfig.resolve.alias['@redwoodjs/router$'] = path.join(getPaths().base, 'node_modules/@redwoodjs/testing/dist/MockRouter.js')
-    sbConfig.resolve.alias['~__REDWOOD__USER_ROUTES_FOR_MOCK'] = getPaths().web.routes
+    sbConfig.resolve.alias['@redwoodjs/router$'] = path.join(
+      getPaths().base,
+      'node_modules/@redwoodjs/testing/dist/MockRouter.js'
+    )
+    sbConfig.resolve.alias[
+      '~__REDWOOD__USER_ROUTES_FOR_MOCK'
+    ] = getPaths().web.routes
```

Comment:
Prettier did this?

Reply:
I think so, I definitely didn't do it manually. Happy to revert it if you want; not getting a linting error in my editor right now the way it is. Could I have configured the settings differently somehow?

Reply:
Tried putting it back to how it was and I wasn't getting a linting error, so reverted it!